style.css

```diff
@@ -1,46 +1,41 @@
 .subscribe{
 	margin: 0 auto;
```

Comment:
should just be `margin: 0`. Written like this, `margin-top` and `margin-bottom` are set to `auto`, which you don't want.